map2loop/project.py

```diff
@@ -204,18 +207,12 @@ def __init__(
         if fault_orientation_filename != "":
             self.map_data.set_filename(Datatype.FAULT_ORIENTATION, fault_orientation_filename)
 
-        if config_filename != "":
```

Comment:
Do we want to remove the capability to load a config file from json? I would probably prefer to allow the config dictionary to be a path/string and be loaded using json.